src/configuration/iconfiguration.ts

```diff
@@ -79,6 +79,21 @@ export interface ICamelCaseMotionConfiguration {
   enable: boolean;
 }
 
+export interface ISmartQuotesConfiguration {
```

Comment:
Same changes as requested in `package.json`

Reply:
changes were committed